sw/device/lib/dif/dif_rv_timer.c

```diff
@@ -0,0 +1,194 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_rv_timer.h"
+
+#include <stddef.h>
+
+#include "rv_timer_regs.h"  // Generated.
+
+/// Converts an offset for a per-hart register at hart 0 to an offset
```

Comment:
Could you please use doxygen style comments to be consistent with DIFs that are already in. In this case:

```
/**
* Converts an offset for a per-hart register at hart 0 to an offset
* at hart |hart|.
*/
```
The same for other cases.

Reply:
Done.